core/shared/src/main/scala/monocle/syntax/ApplyFold.scala

```diff
@@ -67,3 +61,22 @@ case class ApplyFold[S, A](s: S, _fold: Fold[S, A]) {
   /** alias to composeIso */
   @inline def ^<->[B, C, D](other: PIso[A, B, C, D]): ApplyFold[S, C] = andThen(other)
 }
+
+object ApplyFold {
+  implicit def applyFoldSyntax[S, A](self: ApplyFold[S, A]): ApplyFoldSyntax[S, A] =
+    new ApplyFoldSyntax(self)
+}
+
+final case class ApplyFoldSyntax[S, A](self: ApplyFold[S, A]) extends AnyVal {
```

Comment:
what are the benefits of defining `ApplyFoldSyntax`? `Fold` and `Getter` are monomorphic (2 type parameters instead of 4), so I am not sure we need these syntax classes.

Reply:
yes, you are right, as they are monomorphic, that does not bring any benefits except for consistency with the neighboring objects. i'll revert the change, it was mostly experimental :))